roles/ceph-validate/tasks/check_devices.yml

```diff
@@ -1,4 +1,26 @@
 ---
+- name: find device used for operating system
+  command: findmnt -n -T / -o SOURCE
```

Comment:
Just wondering if the could just use the `ansible_mounts` fact there. wdyt ?

Reply:
that is possible indeed, but we would have to resolve the device returned by the fact by using `readlink -f` anyway

Reply:
using `ansible_mounts` makes it a bit more complex since we have to register a variable in a loop to find which one is mounted on `/` ...